src/.nuget/toolchain.ubuntu.14.04-arm.Microsoft.DotNet.ILCompiler.Development.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<package >
```

Comment:
The dotnet restore should pick the right host architecture because of the rid. So the root package can just be ILCompiler.SDK.Arm64.